### PR TITLE
"Competitive Example" (2.5): Sequence-to-sequence

### PR DESCRIPTION
**Pull request by gpengzhi**
Here is a comparison between Dynet and PyTorch on the seq2seq translator example used in PyTorch tutorial (https://pytorch.org/tutorials/intermediate/seq2seq_translation_tutorial.html).

### CODE REVIEW

**Comment by xunzhang** [comment]
there are something wrong with this pull request, I'll help fix this tmr.

**Comment by shuheik** [comment]
Just a question. Why _dynet.py and _pytorch.py use different learning rates?

**Comment by gpengzhi** [comment]
When I tried the learning rate the PyTorch example used (0.01), the performance is very bad. I then choose a reasonable good learning rate for the Dynet example. When I selected the learning rate 0.2 in PyTorch example, it achieved very bad performance.

**Comment by shuheik** [comment]
@gpengzhi Thank you for the explanation.
